packages/libs/react-ui/src/components/Logo/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { KadenaLogo } from './variants/Kadena';
+import { KadenaDevToolsLogo } from './variants/KadenaDevTools';
+import { KadenaDocsLogo } from './variants/KadenaDocs';
+
+import React, { FC, SVGProps } from 'react';
+
+export type LogoVariant = 'default' | 'DevTools' | 'Docs';
+export const logoVariants = ['default', 'DevTools', 'Docs'] as LogoVariant[];
+
+const renderSwitch = (
+  logo: LogoVariant = 'default',
+): FC<SVGProps<SVGSVGElement>> => {
+  switch (logo) {
+    case 'Docs':
+      return KadenaDocsLogo;
+    case 'DevTools':
+      return KadenaDevToolsLogo;
+    default:
+      return KadenaLogo;
+  }
+};
+
+const Logo: FC<{ variant?: LogoVariant }> = ({ variant }) => {
```

Comment:
I was thinking we could extend it later though to include Marmalade logo and so on. What do you think? Otherwise yeah, makes sense to change it to be more explicit.

Reply:
Ahh ok. Maybe `BrandLogo`?

Reply:
Sounds good! Done :)